v0/destinations/util.js

```diff
@@ -66,6 +53,7 @@ const updatePayload = (currentKey, eventMappingArr, value, payload) => {
     if (obj.rudderKey === currentKey) {
```

Comment:
Instead of map, we should be using a while/for-each construct here.

Reply:
Done